nuclia/lib/nua.py

```diff
@@ -52,7 +52,12 @@
     Tokens,
 )
 from nuclia_models.predict.remi import RemiRequest, RemiResponse
+import os
+from tqdm import tqdm
+import asyncio
 
+MB = 1024 * 1024
+CHUNK_SIZE = 10 * MB
```

Comment:
I think intermediate chunk size has to be multiple of some value on aws.

Reply:
but this is upload to our api, that chunking you say it's handled in our code already